backend/src/main/java/com/example/backend/sett/messages/MessageService.java

```diff
@@ -16,7 +15,11 @@ public List<OneMessage> getMessages() {
         return messageRepo.findAll();
     }
 
-    public String getRandomId() {
-        return UUID.randomUUID().toString();
+    public boolean deleteMessage(String id) {
+        if (messageRepo.existsById(id)) {
+            messageRepo.deleteById(id);
+            return true;
+        }
+        return false;
```

Comment:
Du könntest auch eine Möglichkeit überlegen hier eine custom Exception zu throw, wie in GastVortrag, dann könntest du auch Nachricht für üser anpassen

Reply:
Danke war eine gute Idee